src/my-requests.tsx

```diff
@@ -208,114 +210,180 @@ const ProceedingRow = (props: ProceedingRowProps) => {
 
     const original_request = findOriginalRequest(props.proceeding);
 
+    const initialMessage = useMemo<Omit<Message, 'id'>>(
+        () => ({
+            transport_medium: 'email',
+            date: new Date(),
+            reference: props.proceeding.reference,
+            sentByMe: false,
```

Comment:
If we wanted to be extra cool, it might be nice to alternate this depending on what the last message in the proceeding was (assuming that there's typically a back-and-forth).